rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -403,30 +403,44 @@ namespace rsx
 			const u32 cmd = ReadIO32(get);
 			const u32 count = (cmd >> 18) & 0x7ff;
 
-			if (cmd & CELL_GCM_METHOD_FLAG_JUMP)
+			if ((cmd & RSX_METHOD_OLD_JUMP_CMD_MASK) == RSX_METHOD_OLD_JUMP_CMD)
 			{
-				u32 offs = cmd & 0x1fffffff;
+				u32 offs = cmd & 0x1ffffffc;
 				//LOG_WARNING(RSX, "rsx jump(0x%x) #addr=0x%x, cmd=0x%x, get=0x%x, put=0x%x", offs, m_ioAddress + get, cmd, get, put);
 				ctrl->get = offs;
 				continue;
 			}
-			if (cmd & CELL_GCM_METHOD_FLAG_CALL)
+			if ((cmd & RSX_METHOD_NEW_JUMP_CMD_MASK) == RSX_METHOD_NEW_JUMP_CMD)
+			{
+				u32 offs = cmd & 0xfffffffc;
+				//LOG_WARNING(RSX, "rsx jump(0x%x) #addr=0x%x, cmd=0x%x, get=0x%x, put=0x%x", offs, m_ioAddress + get, cmd, get, put);
+				ctrl->get = offs;
+				continue;
+			}
+			if ((cmd & RSX_METHOD_CALL_CMD_MASK) == RSX_METHOD_CALL_CMD)
```

Comment:
No. One of those is CALL_CMD_MASK, which is the mask i have to use to help determine if this is a call command.